tests/integration/goldens/asset/samples/generated_samples/snippet_metadata_v1_asset_v1.json

```diff
@@ -0,0 +1,1048 @@
+{
+  "snippets": [
+    {
+      "clientMethod": {
+        "async": true,
+        "method": {
+          "fullName": "AnalyzeIamPolicyLongrunning",
```

Comment:
fullName shuld be the RPC method name qualified by the proto package and the service name
This looks more like shortName.

Reply:
Ack, I'll switch this out for `shortName`.